packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -1016,6 +1016,20 @@ const api = new apigateway.SpecRestApi(this, 'ExampleRestApi', {
 [`x-amazon-apigateway-endpoint-configuration`](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-swagger-extensions-endpoint-configuration.html) 
 in your openApi file. 
 
+### Permissions
```

Comment:
Move this section to around the line 240 just above the section 'Working with Models' and add a corresponding entry in the table of contents at the top.